buildSrc/src/main/java/quilt/internal/tasks/diff/CheckUnpickVersionsMatchTask.java

```diff
@@ -12,11 +12,11 @@
 import org.gradle.api.tasks.TaskAction;
 import quilt.internal.tasks.DefaultMappingsTask;
 
-public class CheckUnpickVersionsMatchTask extends DefaultMappingsTask {
+public abstract class CheckUnpickVersionsMatchTask extends DefaultMappingsTask {
     public static final String TASK_NAME = "checkUnpickVersionsMatch";
 
     @InputFile
-    private final RegularFileProperty unpickJson;
+    public abstract RegularFileProperty getUnpickJson();
 
     @Internal
     private boolean match = false;
```

Comment:
Same as my worry with targetVersion above -- I'm not quite sold on this approach, it seems like a _bit_ of a bad pattern.